### DISCUSSION TITLE
Support OCIRepository as a helm sourceRef

### DISCUSSION THREAD

**Discussion by willejs** [fluxcd/flux2#4000] [Proposals]
Currently you can use a OCIRepository as a kustomization sourceRef. This is great, as i can package my helmRelease resources as oci containers and deploy them to the cluster, without using a git repository as a source.
Ideally I would be able to use the OCIRepository as a helm source too and then reference to the helm charts inside it without the need to build and publish a chart?
Does this sound feasible?

**Comment by makkes** [comment]
Related issue: https://github.com/fluxcd/helm-controller/issues/616
